integration/test_qca.in

```diff
@@ -0,0 +1,6 @@
+(FIND-QCA-PATH :TARGET ("SHC1") :SOURCE ("IRS1") :RELTYPE ("controls-state-change-of"))
+(SUCCESS :paths (((IRS1 ((:directed True :interaction controls-state-change-of)) HRAS ((:directed True :interaction controls-state-change-of)) RHOA ((:directed True :interaction controls-state-change-of)) SHC1) (IRS1 ((:directed True :interaction controls-state-change-of)) PIK3CA ((:directed True :interaction controls-state-change-of)) RHOA ((:directed True :interaction controls-state-change-of)) SHC1))))
+(HAS-QCA-PATH :TARGET ("SHC1") :SOURCE ("IRS1"))
+(SUCCESS :haspath (True))
+(FIND-QCA-PATH :TARGET ("CALM1") :SOURCE ("NFATC2"))
+(SUCCESS :paths (((IRS1 (:108 (:interaction in-complex-with)) SHC1) (IRS1 (:93584068 (:interaction in-complex-with)) SHC1))))
```

Comment:
I don't think this last test makes sense here - shouldn't it be a path between NFATC2 and CALM1?
I got this when running the system: (SUCCESS :paths (((NFATC2 ((:directed True :interaction controls-expression-of)) IFNG ((:directed True :interaction controls-state-change-of)) CALM1) (NFATC2 ((:directed True :interaction controls-expression-of)) CXCL8 ((:directed True :interaction controls-state-change-of)) GNA14 ((:directed True :interaction controls-state-change-of)) CALM1))))